### PR TITLE
Improve FLUX tool for output label and prompt display

### PR DESCRIPTION
**Pull request by arash77**
Enhance the output label based on the input type and include the prompt in the console output when using FLUX tool.

### CODE REVIEW

File: tools/flux/flux.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@ python3 '$__tool_directory__/flux.py'
         </conditional>
     </inputs>
     <outputs>
-        <data name="output" format="png" label="${tool.name} on ${on_string}" from_work_dir="./output.png"/>
+        <data name="output" format="png" label="${tool.name} on ${on_string if $input_type_selector=='file' else 'text prompt'}" from_work_dir="./output.png"/>
```
Comment (bgruening) [inline tools/flux/flux.xml:42]: if that a good idea? We don't know how large the prompt will be

Comment (arash77) [reply tools/flux/flux.xml:42]: It doesn't add the prompt it self, it just announces that it is a text prompt.
Before it was empty, if it was not a file.

Comment (arash77) [reply tools/flux/flux.xml:42]: Or should it be a short text about the prompt?

Comment (bgruening) [reply tools/flux/flux.xml:42]: No, that is ok. Thanks

**Pull request merged**